exotations/solvers/exotica_ompl_solver/src/ompl_solver.cpp

```diff
@@ -210,30 +206,30 @@ void OMPLSolver<ProblemType>::Solve(Eigen::MatrixXd &solution)
 
     if (!state_space_->as<OMPLStateSpace>()->isLocked())
     {
-        state_space_->as<OMPLStateSpace>()->setBounds(prob_);
-        bounds_ = prob_->getBounds();
+        state_space_->as<OMPLStateSpace>()->SetBounds(prob_);
+        bounds_ = prob_->GetBounds();
     }
-    else if (!bounds_.empty() && bounds_ != prob_->getBounds())
+    else if (!bounds_.empty() && bounds_ != prob_->GetBounds())
     {
-        throw_pretty("Cannot set new bounds on locked state space!");
+        ThrowPretty("Cannot set new bounds on locked state space!");
     }
 
     ompl_simple_setup_->getSpaceInformation()->setup();
 
     ompl_simple_setup_->setup();
 
     if (ompl_simple_setup_->getPlanner()->params().hasParam("range"))
-        ompl_simple_setup_->getPlanner()->params().setParam("range", init_.Range);
+        ompl_simple_setup_->getPlanner()->params().setParam("range", init_.range);
     if (ompl_simple_setup_->getPlanner()->params().hasParam("goal_bias"))
-        ompl_simple_setup_->getPlanner()->params().setParam("goal_bias", init_.GoalBias);
+        ompl_simple_setup_->getPlanner()->params().setParam("goal_bias", init_.goal_bias);
 
-    if (init_.RandomSeed != -1)
+    if (init_.random_seed != -1)
     {
-        HIGHLIGHT_NAMED(algorithm_, "Setting random seed to " << init_.RandomSeed);
-        ompl::RNG::setSeed(init_.RandomSeed);
+        HIGHLIGHT_NAMED(algorithm_, "Setting random seed to " << init_.random_seed);
+        ompl::RNG::setSeed(init_.random_seed);
     }
 
-    SetGoalState(prob_->getGoalState(), init_.Epsilon);
+    SetGoalState(prob_->GetGoalState(), init_.epsilon);
```

Comment:
Epsilon also changes from upper to lower case. In this case, though, it is an improvement (as for alpha in the IMesh taskmap) as it fixes a previously incorrect case.